command/config/config.go

```diff
@@ -363,6 +363,11 @@ type EnvConfig struct {
 	Database struct {
 		Driver     string `envconfig:"DRONE_DATABASE_DRIVER" default:"sqlite3"`
 		Datasource string `envconfig:"DRONE_DATABASE_DATASOURCE" default:"database.sqlite3"`
+		Postgres   struct {
```

Comment:
Can we move this struct out to `Distributed`? Maybe something like
```
   DistributedMode {
        Enabled
        Driver
        Datasource
   }
```

Reply:
Sure

Reply:
Done